### DISCUSSION TITLE
Consider adding extension point on the "info" workspace view

### DISCUSSION THREAD

**Discussion by enkelmedia** [umbraco/Umbraco-CMS#16687] [Features and ideas]
**Is your feature request related to a problem? Please describe.**
I know that it's possible to append new workspaceViews to the content/media section but I can see that it would be valuable to be able to mount a extension on the in the info-view:

![image](https://github.com/umbraco/Umbraco.CMS.Backoffice/assets/1782524/1c0db355-c516-4b65-84f6-67ca87ab0a07)

This could be similar to the `userGranularPermission` extension point, just making it possible to mount a element on the info-view.

**Describe the solution you'd like**
It would be very useful for things that might not be important enough to require it's own icon/workspaceView. For example we have a internal extension where editors can create redirects to a given page (similar to how Skybrud Redirects works) and it would be quite cool to show a list of all redirects pointing to the current page and also make it possible to create a new redirect from the info-view.

**Describe alternatives you've considered**
workspaceView would be a possible solution but I think that 

What do you think?

**Comment by madsrasmussen** [comment]
I would like this too.  I even think that the URL and history should be extensions too 😃 

It could be interesting to see if we could come up with how the extension should work. Would it be a specific extension for documents or could it make sense to make it more generic and do some filtering on entity type? I am not 100% sure how many workspaces we have the info view on these days.

A name suggestion could be something like "workspace info app" to follow the same convention as other extensions where the UI can be anything.

**Comment by enkelmedia** [comment]
Hi!

Great idea, if this was an extension point it would make very much sense for URL and history to also be extensions.

It would be great to have this as a extension point on media and members as well, like you say, it could be the same "thing" and then we could use conditions to control where it would show up I guess? The extension point could even be reused if a package provide a workspaceView similar to the info-view.

I like the name "workspace info app"! =D

// Markus

**Comment by madsrasmussen** [comment]
I have opened a PR for this feature: https://github.com/umbraco/Umbraco-CMS/pull/18014

**Comment by madsrasmussen** [comment]
This extension point will be available in 15.3

**Comment by enkelmedia** [comment]
Thats awesome!! Great stuff @madsrasmussen 🎉
